components/footer.tsx

```
@@ -0,0 +1,101 @@
+          </p>
+        </div>
+      </div>
+      <hr className="h-1 bg-(--omori-black)" />
+      <div className="flex flex-col gap-4">
+        <div className="flex flex-col md:flex-row items-center justify-between w-full gap-2">
+          <p className="text-sm text-center md:text-left text-wrap">
+            © 2025 OMORI Headspace Made With 💜 By{" "}
+            <a
+              href="https://www.github.com/florixak"
+              target="_blank"
+              className={
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)